ttnn/cpp/ttnn/operations/data_movement/slice/slice_pybind.hpp

```diff
@@ -16,27 +16,33 @@ namespace py = pybind11;
 
 void bind_slice(py::module& module) {
     auto doc =
-        R"doc(
-            slice(input_tensor: ttnn.Tensor, slice_start: List[int[tensor rank], slice_end: List[int[tensor rank],  value: Union[int, float], *, Optional[ttnn.MemoryConfig] = None) -> ttnn.Tensor
+        R"doc(slice(input_tensor: ttnn.Tensor, slice_start: List[int[tensor rank], slice_end: List[int[tensor rank], slice_end: List[int[tensor rank], memory_config: Optional[MemoryConfig] = std::nullopt, queue_id: int = 0) -> ttnn.Tensor
 
             Returns a sliced tensor. If the input tensor is on host, the slice will be performed on host, and if its on device it will be performed on device.
 
-            Equivalent pytorch code:
+            Args:
+                input_tensor: Input Tensor.
+                slice_start: Start indices of input tensor. Values along each dim must be < input_tensor_shape[i].
+                slice_end: End indices of input tensor. Values along each dim must be < input_tensor_shape[i].
+                slice_step: (Optional[List[int[tensor rank]]) Step size for each dim. Default is None, which works out be 1 for each dimension.
 
-            .. code-block:: python
+            Keyword Args:
+                memory_config Memory Config of the output tensor
+                queue_id (Optional[uint8]) command queue id
```

Comment:
Please see how we mark optional arguments, it should be `(uint8, optional)` if I am not mistaken.

Reply:
Done, copied the one from permute which is uint8, optional like you said